src/app/InteractionModelEngine.h

```diff
@@ -657,12 +677,15 @@ class InteractionModelEngine : public Messaging::UnsolicitedMessageHandler,
 #endif // CHIP_CONFIG_PERSIST_SUBSCRIPTIONS && CHIP_CONFIG_SUBSCRIPTION_TIMEOUT_RESUMPTION
 #endif // CONFIG_BUILD_FOR_HOST_UNIT_TEST
 
-#if CHIP_CONFIG_PERSIST_SUBSCRIPTIONS && CHIP_CONFIG_SUBSCRIPTION_TIMEOUT_RESUMPTION
+#if CHIP_CONFIG_PERSIST_SUBSCRIPTIONS
+    int8_t mNumOfSubscriptionToResume = 0;
```

Comment:
This member could benefit from some documentation explaining what it's counting... It does not seem to count "resume attempts" in the sense that it gets decremented partway through the attempt.  What is it actually tracking?

Reply:
done